### PR TITLE
#5521 - Fix color contrast

### PR DESCRIPTION
**Pull request by elemdos**
### Details
Fixes several accessibility color contrast issues throughout the app

### Fixed Issues

$ https://github.com/Expensify/App/issues/5521

### Tests
N/A, styling update

### QA Steps
Verify all inputs, checkboxes, and dropdowns are visible & pass contrast requirements. 

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
image

#### Mobile Web
Simulator Screen Shot - iPhone 12 - 2021-10-13 at 13 11 14

#### Desktop
Screen Shot 2021-10-13 at 1 15 16 PM

#### iOS
Simulator Screen Shot - iPhone 12 - 2021-10-13 at 12 48 03

#### Android
Screen Shot 2021-10-13 at 1 04 21 PM

### CODE REVIEW

**Comment by parasharrajat** [comment]
@mateomorris Could you please share a recording of going through all the common pages and design elements so that we can make sure? And it will help the design team as well.

**Comment by Beamanator** [comment]
Here's a video I took on this fork:

https://user-images.githubusercontent.com/3885503/137225559-ebdfc82c-09e4-4996-bcb3-f8c4024ac648.mov

The only slight concern is around dropdowns: the border is blue when focused and light gray when not focused, but I think it should be dark gray when not focused and black when focused, like the input fields right?

Screen Shot 2021-10-13 at 4 20 39 PM

**Comment by shawnborton** [comment]
I think for both text inputs and dropdowns/selects, we want the focus border color to be blue.

I had a short day today, but I'll give this a full review with my feedback tomorrow morning!

**Comment by shawnborton** [comment]
So yes as @Beamanator pointed out, the big thing I am noticing from his video is that the select menus aren't styled the same (border color, active border color) as text inputs. Could you take a look at that for us?

**Comment by mateomorris** [comment]
@shawnborton not sure, but that's how it is on new.expensify so it's not caused by these changes (just the darker pin seems to be)

image

**Comment by shawnborton** [comment]
So it looks like the input border color is too dark:

image

File: src/styles/themes/default.js [hunk 1]
```diff
@@ -11,12 +11,13 @@ export default {
     heading: colors.dark,
     sidebar: colors.gray1,
     sidebarHover: colors.white,
-    border: colors.gray2,
     borderFocus: colors.blue,
-    icon: colors.gray3,
+    componentBorder: colors.gray5,
```
[the remark this answers is not in the document]
Comment (mateomorris) [reply src/styles/themes/default.js]: ah my mistake, fixed

**Comment by shawnborton** [comment]
Unless any changes @mateomorris made caused that regression to happen, I think we can handle that elsewhere @Beamanator. 

Overall I think this is in a good spot. The next step is that I want to quickly show these changes internally before we merge this PR just to make sure everyone is aware of the color/branding implications of changing these items. I will keep you posted @mateomorris and hopefully we can get this merged tomorrow. Thanks!

**Pull request closed without merging**